src/components/sbb-navigation/sbb-navigation.stories.js

```diff
@@ -15,18 +15,17 @@ const playStory = async ({ canvasElement }) => {
   );
 
   const button = canvas.getByTestId('navigation-trigger');
-  await userEvent.click(button);
-  await waitFor(() =>
-    expect(
-      canvas.getByTestId('navigation').classList.contains('sbb-navigation--opened')
-    ).toBeTruthy()
-  );
+  userEvent.click(button);
```

Comment:
I'd suggest to leave the check in this method as it's like a unit test asserting that navigation was really opened.